recipe/meta.yaml

```diff
@@ -1,34 +1,29 @@
-{% set name = "flash-attn" %}
 {% set version = "2.6.3" %}
 
 package:
-  name: {{ name|lower }}
+  name: flash-attn-split
   version: {{ version }}
 
 source:
-  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/flash_attn-{{ version }}.tar.gz
+  - url: https://pypi.io/packages/source/f/flash-attn/flash_attn-{{ version }}.tar.gz
     sha256: 5bfae9500ad8e7d2937ebccb4906f3bc464d1bf66eedd0e4adabd520811c7b52
   # Overwrite with a simpler build script that doesn't try to revend pre-compiled binaries
   - path: pyproject.toml
   - path: setup.py
 
 build:
-  number: 1
+  number: 2
   script: {{ PYTHON }} -m pip install . -vvv --no-deps --no-build-isolation
   script_env:
     # Limit MAX_JOBS in order to prevent runners from crashing
     - MAX_JOBS=4
-    - TORCH_CUDA_ARCH_LIST=8.0;8.6;8.9;9.0+PTX
+    - TORCH_CUDA_ARCH_LIST=8.0+PTX
```

Comment:
Guessing this is just to help CI while we are testing/iterating. Just wanted to flag it so we remember to change it back later